### PR TITLE
Uploads

### PR DESCRIPTION
**Pull request by nbumbarger**
Note: This requires an S3 bucket configured to allow `GET` and `PULL` requests from the [uploader front end's](https://github.com/hotosm/oam-uploader) domain (represented by `*` in the example below), which must include a provision to allow `Content-type` headers.
```
<?xml version="1.0" encoding="UTF-8"?>
<CORSConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
    <CORSRule>
        <AllowedOrigin>*</AllowedOrigin>
        <AllowedMethod>GET</AllowedMethod>
        <AllowedMethod>PUT</AllowedMethod>
        <MaxAgeSeconds>3000</MaxAgeSeconds>
        <AllowedHeader>Authorization</AllowedHeader>
        <AllowedHeader>Content-type</AllowedHeader>
    </CORSRule>
</CORSConfiguration>
```
@danielfdsilva Could you have a look, in combination with the changes to the [uploader front end](https://github.com/hotosm/oam-uploader/pull/65)?

### CODE REVIEW

**Comment by nbumbarger** [comment]
@smit1678 I assume that this is failing the processing test due to the CORS requirement described above, if you could update the bucket settings.

File: routes/uploads.js [hunk 1]
```diff
@@ -73,6 +81,39 @@ module.exports = [
...
+      var params = {
+        Bucket: config.oinBucket,
+        Key: payload.name,
+        ContentType: payload.type,
+        Expires: 60
+      };
+      s3.getSignedUrl('putObject', params, function (err, url) {
+        if (err) {
+          console.log(err);
+          return reply({code: 500, url: null});
```
Comment (danielfdsilva) [inline routes/uploads.js]: @nbumbarger you should use `Boom` to ensure the responses are consistent.

**Comment by smit1678** [comment]
@nbumbarger What's next here? Are the changes @danielfdsilva is requesting blocking? Before we merge this, we should merge in #77 and #76 and do a final test.

**Comment by smit1678** [comment]
To update here, #77 and #76 have been merged in and tested. Requested change isn't blocking but a change that will be implemented. If it can't be implemented today, I'm going to proceed with merging and getting this new work live.

**Comment by smit1678** [comment]
Thanks @nbumbarger for posting #78. We'll come back to this after we merge and go live. Going to get this merged, tested, and pushed to production this weekend.

**Pull request merged**